mysql-test/suite/tianmu/r/issue1251.result

```diff
@@ -0,0 +1,3 @@
+select (select version() regexp '^5.7.36-StoneDB-v[0-9]*\\.[0-9]*\\.[0-9]*$') or (select version() regexp '^5.7.36-StoneDB-v[0-9]*\\.[0-9]*\\.[0-9]*\\.[0-9a-z]{9}$');
```

Comment:
if we upgrade our core to `5.7.4x`, what will happen to this line code.

Reply:
the sql result wil be 0, need to modify the testcase if upgrade to 5.7.4x